pthread.h not found on Windows (using MSVC)

When running `cargo build` on Windows using MSVC (via Github Actions), I'm getting this error:

```
  running: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.27.29110\\bin\\HostX64\\x64\\cl.exe" "/nologo" "/MD" "/Z7" "/I" "D:\\a\\vorbis-sys\\vorbis-sys\\libvorbis\\include" "/I" "D:\\a\\vorbis-sys\\vorbis-sys\\libvorbis\\lib" "/I" "C:\\Users\\runneradmin\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\ogg-sys-0.0.9\\libogg\\include" "/D_USRDLL" "/DLIBVORBIS_EXPORTS" "/W4" "/FoD:\\a\\vorbis-sys\\vorbis-sys\\target\\debug\\build\\vorbis-sys-474051d343568d5d\\out\\libvorbis/lib/misc.o" "/c" "libvorbis/lib/misc.c"
  misc.c
  libvorbis/lib/misc.c(14): fatal error C1083: Cannot open include file: 'pthread.h': No such file or directory
  exit code: 2
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -10,8 +10,8 @@ jobs:
         - os: macos-latest
         - os: windows-latest
           toolchain-suffix: -gnu
-        #- os: windows-latest
-        #  toolchain-suffix: -msvc
+        - os: windows-latest
+          toolchain-suffix: -msvc
         - os: ubuntu-latest
     runs-on: ${{ matrix.os }}
     steps:

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -1,0 +1,33 @@
+name: Build and run tests
+on: [push, pull_request]
+env:
+  CARGO_TERM_COLOR: always
+jobs:
+  tests:
+    strategy:
+      matrix:
+        include:
+        - os: macos-latest
+        - os: windows-latest
+          toolchain-suffix: -gnu
+        #- os: windows-latest
+        #  toolchain-suffix: -msvc
+        - os: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable${{ matrix.toolchain-suffix }}
+        override: true
+    - run: |
+        rustup show
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - run: |
+        cargo build
+    - run: |
+        cargo test
+    - run: |
+        cargo package

--- a/libvorbis/build.rs
+++ b/libvorbis/build.rs
@@ -38,7 +38,6 @@ fn main() {
         .file("libvorbis/lib/lsp.c")
         .file("libvorbis/lib/mapping0.c")
         .file("libvorbis/lib/mdct.c")
-        .file("libvorbis/lib/misc.c")
         .file("libvorbis/lib/psy.c")
         .file("libvorbis/lib/registry.c")
         .file("libvorbis/lib/res0.c")

